WcaOnRails/app/models/competition.rb

```diff
@@ -233,6 +238,7 @@ def update_foreign_keys
       CompetitionDelegate,
       CompetitionOrganizer,
       DelegateReport,
+      competition_events,
```

Comment:
Good pickup. Hopefully @timhabermaas will be able to get #381 pushed live soon and we won't have to worry about this anymore :)